tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -65,4 +65,29 @@ Delete VCH and verify
 
     # Check resource pool is removed
     ${ret}=  Run  govc pool.info -json=true host/*/Resources/%{VCH-NAME}
-	Should Contain  ${ret}  {"ResourcePools":null}
+    Should Contain  ${ret}  {"ResourcePools":null}
+
+Attach Disks and Delete VCH
+    # VCH should delete normally during commit/pull/cp/push operations
+    ${output}=  Install VIC Appliance To Test Server
+    Should Contain    ${output}    Installer completed successfully
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${ubuntu}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}=  Run And Return Rc  govc datastore.ls -dc=%{TEST_DATACENTER} %{VCH-NAME}/VIC/
+    Should Be Equal As Integers  ${rc}  0
+
+    # iterate through found images and attach them to the appliance VM
+    ${rc}  ${output}=  Run And Return Rc And Output  for x in $(govc datastore.ls %{VCH-NAME}/VIC/$(govc datastore.ls %{VCH-NAME}/VIC/)/images/); do echo $x; govc vm.disk.attach -disk=%{VCH-NAME}/VIC/$(govc datastore.ls %{VCH-NAME}/VIC/)/images/$x/$x.vmdk -vm.ip=$(govc vm.ip %{VCH-NAME}); done
```

Comment:
actually I'm really happy you're OK with the for loop because that's the most challenging to break out into Robot script